frontend/src/common/components/Card/types.ts

```diff
@@ -4,9 +4,14 @@ interface Card {
   description?: string;
   date?: string;
   links?: string;
-  variant?: string;
+  variant?: CardVariant;
 }
 
 export interface CardProps extends Partial<Card> {
   id: string;
 }
+
+export enum CardVariant {
+  normal = 'normal',
+  primary = 'primary',
+}
```

Comment:
use varaint card style as enum

Reply:
personally, I prefer variant as string union (`'normal' | 'primary'`) as we use typescript it can detect invalid variant also don't need to import enum everywhere